dotnet/src/SemanticKernel/CoreSkills/SemanticFunctionConstants.cs

```diff
@@ -61,49 +77,54 @@ internal static class SemanticFunctionConstants
     description: Get the length of a string.
     inputs:
     - input: Input string
- _GLOBAL_FUNCTIONS_.Hour:
+ _GLOBAL_FUNCTIONS_.HourNumber:
     description: Get the current clock hour
     inputs:
 
+[END EXAMPLE FUNCTIONS]
+
 <goal>If its afternoon please Summarize the input, if the input length > 10 and contains ""book"" then Create an outline for a children's book with 3 chapters about a group of kids in a club and summarize it otherwise translate to japanese and email it to Martin.</goal>
 <plan>
-  <function._GLOBAL_FUNCTIONS_.Hour setContextVariable=""HOUR_NUMBER""/>
+  <function._GLOBAL_FUNCTIONS_.HourNumber setContextVariable=""HOUR_NUMBER""/>
   <if condition=""$HOUR_NUMBER greaterthan 12 OR $HOUR_NUMBER equals 12"">
     <function.Everything.Summarize setContextVariable=""SUMMARIZED_INPUT""/>
     <function._GLOBAL_FUNCTIONS_.Length setContextVariable=""SUMMARIZED_INPUT_LENGTH""/>
     <if condition=""$SUMMARIZED_INPUT_LENGTH greaterthan 10 and $SUMMARIZED_INPUT contains 'book'"">
         <function._GLOBAL_FUNCTIONS_.NovelOutline input=""A group of kids in a club called 'The Thinking Caps' that solve mysteries and puzzles using their creativity and logic."" chapterCount=""3"" />
        <function.Everything.Summarize/>
     </if>
-    <else> (dont use elseif)
+    <else>
         <function.LanguageHelpers.TranslateTo input=""$SUMMARIZED_INPUT"" translate_to_language=""Japanese"" setContextVariable=""TRANSLATED_TEXT"" />
         <function.EmailConnector.LookupContactEmail input=""Martin"" setContextVariable=""CONTACT_RESULT"" />
         <function.EmailConnector.EmailTo input=""$TRANSLATED_TEXT"" recipient=""$CONTACT_RESULT""/>
     </else>
   </if>
 </plan><!-- END -->
+[END EXAMPLE PLANS]
+
+Create an XML plan step by step, to satisfy the goal given.
```

Comment:
@RogerBarreto I'd like to update these rules a bit for the short term to try and get back to a stable place. Let me know if you have thoughts/concerns.